docs/gatsby-config.js

```diff
@@ -36,11 +37,6 @@ module.exports = {
             'advanced/upload',
             'advanced/using-aliases',
           ],
-          'Migration guides': [
```

Comment:
Also figure we don't need to show migration guides for previous versions here anymore